content/installation/java/host/UbuntuHosts.md

```diff
@@ -37,7 +37,9 @@ $ bash -c "java -Dcontrast.stdout=true -version 2>1 | grep Contrast | head -n 1"
 [Contrast] Wed Aug 15 17:37:23 UTC 2018 No TeamServer configuration detected. Agent will only be reporting to local listeners (e.g., Eclipse Plugin).
 ```
 
-* The Exec Helper affects all new shells, as indicated by the `bash -c` in the verification command. Consequently, it will not be present in your current shell. Reload your shell to enable the Exec Helper. Restart any `java` services to enable Contrast for those services.
+* The *contrast-agent.jar* expects a contrast_security.yaml file to be located in /etc/contrast/java/ directory. You can either [download it from the Contrast UI](installation-javastandard.html#contrast-ui) or create it and paste the required [Contrast UI properties](installation-javaconfig.html) 
```

Comment:
It's not entirely accurate to say that the agent requires a YAML file, nor is it accurate to say that the agent requires the YAML file to be at a specific path, because

* users can change the path to the YAML file
* users may use Java system properties instead of a YAML file
* users may use environment variables instead of a YAML file

There's a trade-off here: if we only mention the YAML file, then the instructions are terser, but we may make life difficult for those users that would be better served by system properties and environment variables (e.g. users running their Java web application in containers).

I struggle with which direction to take, but I'm leaning towards showing users how to use the YAML file then mentioning the more in-depth [Agent Configuration](installation-javaconfig.html) page.